extensions/wikia/TriviaQuizzes/TriviaQuizzesHooks.class.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+class TriviaQuizzesHooks {
+    public static function onBeforePageDisplay( OutputPage $out ) {
+        global $wgTitle, $wgEnableTriviaQuizzesExt, $wgTriviaQuizzesEnabledPages;
+
+        if ( $wgEnableTriviaQuizzesExt && in_array( $wgTitle->getText(), $wgTriviaQuizzesEnabledPages ) ) {
```

Comment:
Rather than using the global `$wgTitle`, it would be preferred to use `$out->getTitle()`. Also, `Title::getText` doesn't include the page's namespace, so the titles for both `Project:Foo` and `Foo` would both return `Foo`. As a result, I think using `Title::getPrefixedText` would be better here.

Reply:
Fixed in de8ccb30cf8